src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action=" " 
+      class="search-form"
+      method="get" 
+      data-qa="big">
+      <label class="search-form__label">
+        <input 
+          type="search"
+          class="search-form__input inputBig" 
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+      >
+      </label>
+    </form>
+    <form 
+      action=" " 
+      class="search-form secondSearchBar"
+      method="get"
+      data-qa="small">
```

Comment:
```suggestion
    <form 
      action=" " 
      class="search-form secondSearchBar"
      method="get"
      data-qa="small"
    >
```
открывающий и закрывающий тег должны находиться на одном уровне